locales/en/translation.json

```diff
@@ -227,7 +227,7 @@
       "use-new-object": "If you are making a new object, be sure to use it or store it in a variable",
       "unnecessary-semicolon": "You don't need a semicolon `;` on this line",
       "missing-semicolon": "You need a semicolon `;` at the end of this line",
-      "missing-parentheses": "There should be parentheses here.\nTry: `new {{-object}}()`",
+      "missing-parentheses": "There should be parentheses here.\nTry coding it like this: `new {{-object}}()`",
```

Comment:
I think in this case, we have enough information from the validation error that we can give a suggestion that the students can use as-is?